redaxo/src/addons/structure/plugins/content/lib/api_module.php

```diff
@@ -48,7 +48,7 @@ public function getKey(): string
             $sql->setQuery('select `key` from '. rex::getTable('module') .' where id=?', [$this->module_id]);
 
             if (1 == $sql->getRows()) {
-                $this->key = $sql->getValue('key');
+                $this->key = $sql->getValue('key') ?? '';
```

Comment:
Der Returnwert sollte stattdessen nullable sein, denn hier gibt es ja gerade einen wichtigen Unterschied zwischen null und '', letzteres könnte nicht mehrfach in der DB stehen (wegen Unique index), null schon.

Auch Zeile 45 sollte angepasst werden, und auch rex_template.